package.json

```diff
@@ -82,7 +82,7 @@
     "sass-loader": "^8.0.2",
     "source-map-loader": "^1.0.0",
     "style-loader": "^1.2.1",
-    "stylelint": "^13.5.0",
+    "stylelint": "^13.6.0",
```

Comment:
Now that I see it, is stylelint even used?

Reply:
There is a script for it, but it's not used in CI and I also do not use it.

Reply:
I wonder if we could just get rid of it

Reply:
I tried it and it actually does nothing = it's removed.